apis/apiextensions/v1/composition_types.go

```diff
@@ -33,42 +35,10 @@ type CompositionSpec struct {
 	// Composition Functions, each of which is responsible for producing
 	// composed resources that Crossplane should create or update.
 	//
-	// "Resources" indicates that a Composition uses what is commonly referred
-	// to as "Patch & Transform" or P&T composition. This mode of Composition
-	// uses an array of resources, each a template for a composed resource.
-	//
-	// All Compositions should use Pipeline mode. Resources mode is deprecated.
-	// Resources mode won't be removed in Crossplane 1.x, and will remain the
-	// default to avoid breaking legacy Compositions. However, it's no longer
-	// accepting new features, and only accepting security related bug fixes.
-	//
 	// +optional
-	// +kubebuilder:validation:Enum=Resources;Pipeline
-	// +kubebuilder:default=Resources
-	Mode *CompositionMode `json:"mode,omitempty"`
```

Comment:
And here, this changes v1 API shape. Intentional?

Reply:
Yeah - related to comment above (same feature).